src/instance.rs

```diff
@@ -519,6 +519,13 @@ impl<T> Py<T> {
         unsafe { ffi::Py_None() == self.as_ptr() }
     }
 
+    /// Returns whether the object is an Ellipsis, e.g. `...`.
```

Comment:
Just a nitpick on the wording - Ellipsis is a singleton so something can't be "an" Ellipsis, it *is* Ellipsis.

Reply:
changed.